Source/Evolve/WarpXComputeDt.cpp

```diff
@@ -27,29 +28,33 @@
 #include <algorithm>
 #include <memory>
 
+AMREX_FORCE_INLINE amrex::Real
+minDim (const amrex::Real* x)
+{
+#if defined(WARPX_DIM_1D_Z)
```

Comment:
I learned a new trick today. This can be simplified to
```
std::min({AMREX_D_DECL(x[0], x[1], x[2])});
```

Reply:
Oh, neat!